components/txn_types/src/lock.rs

```diff
@@ -220,6 +220,14 @@ impl Lock {
         size
     }
 
+    // returns the max size ratio of a Lock to a PessimisticLock, where the lock is obtained by pes_lock.to_lock()
+    pub fn max_size_amplification_from_pes_lock_to_lock() -> f64 {
+        // pes lock: 32 bytes + len(primary)
+        // lock from the pes lock: 59 bytes + len
```

Comment:
1. what the '59 bytes' is made up? 
<img width="447" alt="image" src="https://user-images.githubusercontent.com/87681388/183357122-9d031f33-298c-4907-b35c-5ccd94aa14e3.png">